portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,67 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//Training of fecth()
+async function addGreetingElmer(){
+
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+
+    const dateContainer = document.getElementById('greeting-container');
+    dateContainer.innerText = textFromResponse;
+    
+}
+/** Fetches stats from the server and adds them to the page. */
```

Comment:
nit: incorrect comment. Also see other places

If you feel like a comment isn't necessary it's fine to omit it, but better to not have incorrect comments :)

Reply:
Sorry for that I copy the example class, I was overherlmed doing the function and forgot change the comments. Sorry for that. When I will do the part 2 pull request I will correct the comments